### PR TITLE
[WIP] 10-minute tutorial with Koalas

### PR DESCRIPTION
**Pull request by garawalid**
In this tutorial, we will cover the basics of Koalas.

This PR follows up #34

### CODE REVIEW

**Comment by garawalid** [comment]
@dennyglee 
We can start in this PR. Let's focus on Viewing Data as @thunterdb suggested.

@thunterdb 
Please feel free to comment.

**Comment by dennyglee** [comment]
Thanks!  I'll review this and start working on this in the next day or two.  Rock on!

**Comment by thunterdb** [comment]
@garawalid thanks! Can you remove for now all the other sections? It is easier to review.

**Comment by garawalid** [comment]
@thunterdb I will keep only two sections for the moment, Object Creation & Viewing Data. 
We will let you know once we have a clean draft.

**Comment by dennyglee** [comment]
@garawalid Which sections would you like me to focus on tonight/tomorrow?

**Comment by garawalid** [comment]
@dennyglee You can focus on `Selection`. Also, feel free to leave me some comments in the notebook for the two other sections. 
I'll take `Missing Data` and `Operations`

File: databricks/koalas/typedef.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 """
 Utilities to deal with types. This is mostly focused on python3.
 """
+
```
Comment (rxin) [inline databricks/koalas/typedef.py]: can you remove this unrelated change?

Comment (garawalid) [reply databricks/koalas/typedef.py]: Thanks for pointing out this mistake. 
I'll rebase this PR.

**Comment by HyukjinKwon** [comment]
It's merged at https://github.com/databricks/koalas/commit/d8c7f73b88ad84f28dd69d5687b847654af4b04e - @garawalid was co-authored.
